### PR TITLE
fixed: wrong props to SDP onClose(#1674)

### LINKED ISSUE #1674: SingleDatePicker: startDate and endDate passed to onClose when clicking outside
react-dates@20.2.0

While implementing the `SingleDatePicker` I found out that `onDayClick` passes `{ date: Moment }` and `onOutsideClick` passes `{ startDate: undefined, endDate: undefined }` to `onClose`. See: https://github.com/airbnb/react-dates/blob/11de2016bf6e7111ddebd5a7f7fc1fe929a3600c/src/components/SingleDatePicker.jsx#L206

Should the latter even be the case for `SingleDatePicker`, since there's only one date?

### PR DESCRIPTION
**Pull request by AlokTakshak**
Fixes #1674.

### CODE REVIEW

File: src/components/SingleDatePicker.jsx [hunk 1]
```diff
@@ -203,7 +202,7 @@ class SingleDatePicker extends React.PureComponent {
     });
 
     onFocusChange({ focused: false });
-    onClose({ startDate, endDate });
+    onClose({ date});
```
Comment (ljharb) [inline src/components/SingleDatePicker.jsx]:
```suggestion
    onClose({ date });
```

Comment (AlokTakshak) [reply src/components/SingleDatePicker.jsx]: done

Comment (AlokTakshak) [reply src/components/SingleDatePicker.jsx]: https://github.com/airbnb/react-dates/blob/11de2016bf6e7111ddebd5a7f7fc1fe929a3600c/test/components/SingleDatePicker_spec.jsx#L407
in this test suite should i add a test for `calls props.onFocusChange ` with  `date` props ?

Comment (ljharb) [reply src/components/SingleDatePicker.jsx]: that sounds great, yes

Comment (AlokTakshak) [reply src/components/SingleDatePicker.jsx]: can i run particular test file?

Comment (AlokTakshak) [reply src/components/SingleDatePicker.jsx]: done

**Comment by lencioni** [comment]
Merged in 3290bb4

**Pull request closed without merging**